Editor/Window/AnywherePage.cs

```diff
@@ -10,6 +11,8 @@ namespace AmazonGameLift.Editor
     public class AnywherePage
     {
         private readonly VisualElement _container;
+        private readonly ScriptingDefineSymbolEditor _defineEditor;
+        private const string _unityServerDefine = "UNITY_SERVER";
```

Comment:
We have three separate constants for the same preprocessor symbol. We should move these someone constant (like ScriptingDefineSymbolEditor for now). If we add more symbols in the future it can be a separate file